src/CheckBoxGroup/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ElementPropTypes from 'react-element-proptypes';
+
+import CheckBox from '../CheckBox';
+import styles from './CheckBoxGroup.scss';
+
+function CheckBoxGroup(props) {
+  return (
+    <div className={styles['form-group']}>
+      {props.children}
+    </div>
+  );
+}
```

Comment:
So to be clear here, the primary purpose of the `CheckBoxGroup` is to wrap the checkboxes in a div with the proper styles? Not opposed to this, but curious whether we might want to eventually build additional functionality into this component. Like, would this be the place for something like a `clearAll` method?

Reply:
I could definitely see this component growing in scope, with some sort of `clearAll` method or anything else that I likely haven't thought of yet. But yeah, for the purposes of this PR it was mostly to get the correct stylings in place.